### PR TITLE
Small documentation edits

### PR DESCRIPTION
**Pull request by RichardLitt**
### What are you trying to accomplish with this PR?

I am just adding some small changes to the documentation, making the README easier to parse, and adding fields to the package.json. 

### Checklist
For contributors:
- [x] I have [updated the docs](https://github.com/Shopify/slate/blob/master/CONTRIBUTING.md#documentation) to reflect these changes, if applicable.

For maintainers:
- [ ] I have :tophat:'d these changes.
- [ ] I have bumped the `package.json` version in a separate PR, if applicable.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
   "version": "0.9.7",
   "author": "Shopify Inc.",
   "description": "A theme scaffold and command line tool for developing Shopify themes.",
+  "keywords": ["slate", "shopify", "theme", "CLI"],
+  "homepage": "https://github.com/Shopify/slate",
```
Comment (NathanPJF) [inline package.json]: Any reason to use the GitHub repo over the dedicated homepage? https://shopify.github.io/slate/

**Comment by NathanPJF** [comment]
Thanks for looking this over @RichardLitt - I just have a question about the `homepage` in package.json.

File: package.json (hunk 1 above)
Comment (RichardLitt) [inline package.json]: Because it seems to be the standard for where the code lives, that I've seen on npm. I can change it, if you like?

**Comment by RichardLitt** [comment]
@NathanPJF Edited to include the web page.

**Comment by chrisberthe** [comment]
Thanks for the updates @RichardLitt. @NathanPJF on your side?

**Comment by NathanPJF** [comment]
All good on my side! (Still getting caught up from Unite)

**Pull request merged**